pytket/pytket/__init__.py

```diff
@@ -14,20 +14,20 @@
 
 """Python Interface to tket
 """
+import pytket.architecture  # noqa: I001
```

Comment:
Ruff wanted me to move the line below further down, but for reasons I don't understand this messes with the stubs generation (`Circuit` has to be imported before `mapping` and `placement`).